config/template.config

```diff
@@ -7,47 +7,46 @@ includeConfig "${projectDir}/nextflow.config"
-    // whole genome validation set. Must be in the range [0.0, 1.0].
+    // Optional parameter specifying target Stability Score threshold for variant filtering
+    // Default behavior without `target_threshold` or `target_specificity` uses threshold maximizing F1-score in whole genome validation set
+    // Must be in the range [0.0, 1.0]
     // target_threshold = 0.5
 
-    // Optional parameter specifying target specificity for variant filtering
-    // based on whole genome validation set Overrides `target_threshold`. Must
-    // be in the range [0.0, 1.0],
-    // target_specificity = 0.5
+    // Optional parameter specifying target specificity for variant filtering based on whole genome validation set
+    // Overrides `target_threshold`
+    // Must be in the range [0.0, 1.0]
+    // target_specificity = 0.95
 
-    liftover_direction = "" // Choices: GRCh37ToGRCh38, GRCh38ToGRCh37
+    // Choices: ["GRCh37ToGRCh38", "GRCh38ToGRCh37"]
+    liftover_direction = "GRCh37ToGRCh38"
+    chain_file = "hg19ToHg38.over.chain" // Included in resource-bundle.zip
 
-    // Path to the GRCh37 reference sequence (FASTA).
-    fasta_ref_37 = "/hot/ref/reference/GRCh37-EBI-hs37d5/hs37d5.fa"
-
-    // Path to the GRCh38 reference sequence (FASTA).
-    fasta_ref_38 = "/hot/ref/reference/GRCh38-BI-20160721/Homo_sapiens_assembly38.fasta"
-
-    // The liftover chain file between the source and destination FASTA references
-    // Included in resource-bundle.zip
-    chain_file = "/hot/ref/tool-specific-input/liftOver/hg19ToHg38.over.chain"
+    // Path to reference fasta files
+    fasta_ref_37 = "GRCh37-EBI-hs37d5/hs37d5.fa"
+    fasta_ref_38 = "GRCh38-BI-20160721/Homo_sapiens_assembly38.fasta"
 
     // SNV required references
-    funcotator_data_source = "/hot/ref/tool-specific-input/Funcotator/somatic/funcotator_dataSources.v1.7.20200521s"
-    // Included in resource-bundle.zip
-    repeat_bed = "/hot/ref/database/RepeatMasker-3.0.1/processed/GRCh38/GRCh38_RepeatMasker_intervals.bed"
+    funcotator_data_source = "funcotator_dataSources.v1.7.20200521s_StableLift" // Included in resource-bundle.zip
+    // RepeatMasker intervals file corresponding to target build
+    repeat_bed = "GRCh38_RepeatMasker-intervals.bed" // Included in resource-bundle.zip
 
     // SV required references
-    // Included in resource-bundle.zip, choose header_contigs based on target build
-    header_contigs = "/hot/code/nkwang/GitHub/uclahs-cds/project-method-AlgorithmEvaluation-BNCH-000142-GRCh37v38/report/manuscript/publish/GRCh38-vcf-header-contigs.txt"
-    gnomad_rds = "/hot/code/nkwang/GitHub/uclahs-cds/project-method-AlgorithmEvaluation-BNCH-000142-GRCh37v38/report/manuscript/publish/data/gnomad.v4.0.sv.Rds"
+    // VCF header contigs corresponding to target build
+    header_contigs = "GRCh38_VCF-header-contigs.txt" // Included in resource-bundle.zip
+    gnomad_rds = "gnomad.v4.0.sv.Rds" // Included in resource-bundle.zip
```

Comment:
**issue (blocking):** All of these parameters assume that `resource-bundle.zip` has been unpacked into the current working directory. We should either add that detail to the **How To Run** instructions (which currently just says to download the bundle) _or_ add a `resource_bundle_path` parameter.

Reply:
I was assuming that the user would manage the paths in `template.config` themselves but we could simplify for them. Are you suggesting we add `resource_bundle_path` and then change the defaults to e.g. `gnomad_rds = "${resource_bundle_path}/gnomad.v4.0.sv.Rds"`? Or should it be `${params.resource_bundle_path}`?

Reply:
My concern is with having invalid but valid-looking default data in the template. A line like this...

```groovy
    repeat_bed = "GRCh38_RepeatMasker-intervals.bed" // Included in resource-bundle.zip
```
... would make me think that it has been pre-filled with an acceptable value. I'd be frustrated to realize that I needed to add in the absolute path.

I think your first answer make the most sense:

```groovy
    resource_bundle_path = "" // Path to unpacked resource-bundle.zip

    repeat_bed = "${resource_bundle_path}/GRCh38_RepeatMasker-intervals.bed"
```

Reply:
https://github.com/uclahs-cds/pipeline-StableLift/pull/29/commits/e730615842a1cf1ab1b96513da5968ac17c7cc20: How about this? (can you double check syntax)

Reply:
Looks good!